### ISSUE TITLE
Panic when using SetUseCodepointIndices

### ISSUE THREAD

**Issue by koszuta** [ianlopshire/go-fixedwidth#58]
Version: `v0.9.3`

To recreate the issue, I've boiled it down to:
1. encoding with `SetUseCodepointIndices` enabled
2. have some field which contains a non-ASCII character
3. have some field _at the end_
4. the last field is _right aligned_
5.  the last field is _empty_

```golang
package main

import (
	"bytes"
	"fmt"

	"github.com/ianlopshire/go-fixedwidth"
)

func main() {
	var v struct {
		Foo string `fixed:"1,1"`
		Bar string `fixed:"2,2,right"`
	}
	v.Foo = "Ç"

	buf := new(bytes.Buffer)
	e := fixedwidth.NewEncoder(buf)
	e.SetUseCodepointIndices(true)
	_ = e.Encode(v)
	fmt.Printf("%q\n", buf)
}
```

Output:

```text
panic: runtime error: index out of range [2] with length 2

goroutine 1 [running]:
github.com/ianlopshire/go-fixedwidth.(*lineBuilder).WriteValue(0xc000092ce0, 0x2, {{0x0, 0x0}, {0x0, 0x0, 0x0}})
        /.../go/pkg/mod/github.com/ianlopshire/go-fixedwidth@v0.9.3/buff.go:63 +0x4a5
github.com/ianlopshire/go-fixedwidth.valueEncoder.Write(0x0?, 0x0?, {0x10acd00?, 0xc0000b8010?, 0x0?}, {0x2, 0x2, 0xc0000ac050, 0xc0000ac060, 0x10c7e38, ...})
        /.../go/pkg/mod/github.com/ianlopshire/go-fixedwidth@v0.9.3/encode.go:198 +0x31e
github.com/ianlopshire/go-fixedwidth.structEncoder.func1({0x10b2540?, 0xc0000b8000?, 0x203000?})
        /.../go/pkg/mod/github.com/ianlopshire/go-fixedwidth@v0.9.3/encode.go:244 +0x415
github.com/ianlopshire/go-fixedwidth.(*Encoder).writeLine(0xc000092f30, {0x10b2540?, 0xc0000b8000?, 0x1009c6b?})
        /.../go/pkg/mod/github.com/ianlopshire/go-fixedwidth@v0.9.3/encode.go:146 +0x159
github.com/ianlopshire/go-fixedwidth.(*Encoder).Encode(0xc000092f30, {0x10b2540, 0xc0000b8000?})
        /.../go/pkg/mod/github.com/ianlopshire/go-fixedwidth@v0.9.3/encode.go:112 +0x16f
main.main()
        /.../main.go:20 +0x171
exit status 2
```

Expected output:

```text
"Ç "
```

**Comment by ianlopshire** [comment]
@koszuta, Thanks for submitting with replication steps! I was able to reproduce it locally and will try to track down the issue this evening.

**Comment by ianlopshire** [comment]
@koszuta, I've opened #59 which should fix this issue.

Can you confirm that the issue is resolved for your original use case? If so I will merge and cut a new release.

**Comment by koszuta** [comment]
> @koszuta, I've opened #59 which should fix this issue.
> 
> Can you confirm that the issue is resolved for your original use case? If so I will merge and cut a new release.

Yes, I was able to just remove the right align for my use case. Thanks.

**Comment by ianlopshire** [comment]
Fixed by #59
